crates/ruff/src/rules/tryceratops/rules/try_consider_else.rs

```diff
@@ -17,8 +17,8 @@ impl Violation for TryConsiderElse {
 }
 
 /// TRY300
-pub fn try_consider_else(checker: &mut Checker, body: &[Stmt], orelse: &[Stmt]) {
-    if body.len() > 1 && orelse.is_empty() {
+pub fn try_consider_else(checker: &mut Checker, body: &[Stmt], orelse: &[Stmt], has_handler: bool) {
```

Comment:
Nit: let's pass `handlers`, so that the arguments here roughly match the "shape" of the AST node (and the n check `handlers.is_empty()` in here.

Reply:
Done.